converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Features/Fhir/FhirService.cs

```diff
@@ -82,6 +90,59 @@ private async Task<TResource> SearchByIdentifierAsync<TResource>(Identifier iden
             return await SearchByQueryParameterAsync<TResource>(identifier.ToSearchQueryParameter(), cancellationToken);
         }
 
+
+        private async Task<IEnumerable<TResource>> SearchMultiByIdentifierAsync<TResource>(Identifier identifier, CancellationToken cancellationToken)
```

Comment:
This method and the one below (SearchByQueryParameterAsync) has lots of similarities, can we reuse the  code  and not duplicate it?

Reply:
Refactored in latest commit